MaxText/max_utils.py

```diff
@@ -480,3 +480,11 @@ def init_kv_cache(model, config):
   with mesh, nn_partitioning.axis_rules(config.logical_axis_rules):
     state_mesh_annotations = nn.logical_to_mesh(state_logical_annotations)
   return state_mesh_annotations
+
+def convert_state_to_jnp(state_dictionary):
+  for key, value in state_dictionary.items():
```

Comment:
oh wow this is super dangerous as an impl. can you do this as a `jax.tree_map(lambda x : jnp.array(x), state_dictionary)`?

Reply:
Done - thank you!